build.install4j

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <install4j version="6.0.4" transformSequenceNumber="5">
-  <directoryPresets config="./build/assets" />
```

Comment:
This file is autogenerated? Curious if the remove of "./build/assets" is extra cleanup, or just how install4j works.